orchestration/values.schema.json

```diff
@@ -85,7 +85,24 @@
       },
       "required": ["stagingData", "jadeData"]
     },
-    "maxParallelism": { "type": "integer" }
+    "repo" : {
+      "type": "object",
+      "properties": {
+        "url": {"type":  "string"},
+        "datasetId": {"type": "string"},
+        "profileId": {"type": "string"},
+        "pollTimeout": {"type":  "integer"}
+        },
+      "required": ["url", "datasetId", "profileId", "pollTimeout"]
+    },
+    "maxParallelism": { "type": "integer" },
+    "saCreds": {
```

Comment:
In my head, this would make sense to nest under `repo`. I'd rename to something like `accessKey`, and have the two sub-properties be `secretName` and `secretKey`

Reply:
works for me!